tools/networkmanager/inject_txs.go

```diff
@@ -44,7 +46,9 @@ func InjectTransactions(nmConfig Config) {
 		1*time.Second,
 		stats.NewStats(1),
 		[]ethclient.EthClient{l1Client},
-		[]wallet.Wallet{l1Wallet},
+		&params.SimWallets{
+			// todo
```

Comment:
@tudor-malene Can you expand on this todo?

Reply:
ah yeah, that was actually a question to you. let's chat later about this